torch/export/_trace.py

```diff
@@ -422,6 +437,7 @@ def _export(
     kwargs = kwargs or {}
 
     if not strict:
+        f = _transform_callable_to_nn_module(f)
```

Comment:
I think we should ban exporting free function all together. Could you delete this and also assert that we can't pass in free function to strict export mode as well? 
cc: @zhxchen17 @avikchaudhuri